### ISSUE TITLE
Set up CheXNet for Fine-Tuning

### ISSUE THREAD

**Issue by sebapusch** [sebapusch/X2CT-MMe3D#14]
Set up the DenseNet-121 backbone pretrained on ChestX-ray14 (CheXNet) and adapt it for our binary classification task using frontal and lateral chest X-rays. This will serve as our 2D baseline model.

- [x] Download CheXNet pretrained weights from [arnoweng/CheXNet](https://github.com/arnoweng/CheXNet)  
  Direct download link: https://github.com/arnoweng/CheXNet/raw/master/models/CheXNet.pth

- [x] Load `torchvision.models.densenet121()`

- [x] Replace final classifier with binary classification

- [x] Load weights from `CheXNet.pth`  (Only load weights for matching layers; use `strict=False` if necessary)

- [x] Normalize and prepare dual-view input (stack PA and lateral into 2-channel input)

- [ ] Wrap model for training (move to device, define optimizer, loss)
